packages/ndla-ui/src/LearningPaths/LearningPathInformation.tsx

```diff
@@ -64,15 +65,15 @@ interface Props {
 }
 
 export const LearningPathInformation = ({ description, title, license, invertedStyle, id }: Props) => {
-  const { rights } = getLicenseByAbbreviation(license?.license || "", "nb");
+  const rights = getLicenseByAbbreviation(license?.license || "", "nb");
```

Comment:
Her henter du vel ut heile objectet. Blir meir korrekt å kalle det license.
```suggestion
  const license = getLicenseByAbbreviation(license?.license || "", "nb");
```

Reply:
går ikke for vi allerede får license via props, men jeg er helt enig at det blir dumt å kalle det for rights. Passer det med _fullLicense_?

Reply:
Du kan forsåvidt unngå problemet med å kalle getLicenseByAbbreviation direkte i kallet på LicenseLink.